models/org.go

```diff
@@ -235,37 +237,37 @@ func DeleteOrganization(org *User) (err error) {
 // OrgUser represents an organization-user relation.
 type OrgUser struct {
 	ID       int64 `xorm:"pk autoincr"`
-	Uid      int64 `xorm:"INDEX UNIQUE(s)"`
+	UID      int64 `xorm:"INDEX UNIQUE(s)"`
```

Comment:
The default `GonicMapper` cannot recognize `UID` as `uid` in database. I think you have to add `UID` to the map in https://github.com/go-gitea/gitea/blob/master/models/models.go#L72